src/app/ui/social-page/hof/filter.tsx

```diff
@@ -0,0 +1,100 @@
+'use client'
+
+import React from 'react'
+import { Button, Input } from '@material-tailwind/react'
+import { useDebouncedCallback } from 'use-debounce'
+import { useForm } from 'react-hook-form'
+import { FACULTIES } from '../../../constant'
+import { faFilterCircleXmark } from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+
+export default function Filter({ setMyParams }) {
+  const handleSearch = useDebouncedCallback((keyword) => {})
+
+  const {
+    register,
+    getValues,
+    reset,
+    // eslint-disable-next-line react-hooks/rules-of-hooks
+  } = useForm()
+
+  const handleInputs = (e) => {
+    // const params = new URLSearchParams(searchParams)
+    // params.set(e.target.name, e.target.value)
+    // replace(`${pathname}?${params.toString()}`)
+    // setMyParams(`?status=${status}&${params.toString()}`)
+  }
+  const handleResetForm = () => {
+    // reset()
+    // replace(pathname)
+    // setMyParams(`?status=${status}`)
+  }
+
+  return (
+    <form>
+      <div className="flex flex-wrap lg:flex-nowrap gap-5 max-w-[80rem]">
+        <Input
+          placeholder=""
+          crossOrigin={undefined}
+          size="md"
+          containerProps={{ className: 'col-span-6' }}
+          labelProps={{
+            className: 'before:content-none after:content-none',
```

Comment:
```suggestion
          label="Tìm kiếm..."
```
vì ko có label nên dùng label sẵn của input